src/main/java/gov/cabinetofice/gapuserservice/repository/UserRepository.java

```diff
@@ -1,14 +1,22 @@
 package gov.cabinetofice.gapuserservice.repository;
 
 import gov.cabinetofice.gapuserservice.model.User;
+import org.springframework.data.jpa.repository.EntityGraph;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
 import java.util.Optional;
 
 @Repository
 public interface UserRepository extends JpaRepository<User, Integer> {
+
+    @EntityGraph(attributePaths = {"department", "roles"})
     Optional<User> findByEmail(String email);
+    @EntityGraph(attributePaths = {"department", "roles"})
     Optional<User> findBySub(String sub);
+
+    @EntityGraph(attributePaths = {"department", "roles"})
+    Optional<User> findById(int id);
```

Comment:
Do we defo wanna lazy load if we're gonna eagerly fetch these relations each query anyway?

Reply:
I think I'd rather lazy load as a default and then explicitly eagerly fetch.  Means we are not over fetching in future queries by accident.